scripts/shared/cleanup.sh

```diff
@@ -0,0 +1,41 @@
+
+#!/usr/bin/env bash
+set -em
+
+source ${SCRIPTS_DIR}/lib/debug_functions
+source ${SCRIPTS_DIR}/lib/utils
```

Comment:
Shouldn't the path be _${SCRIPTS_DIR}/shared/lib_/?

Reply:
No since the `shared` library is what we copy into the image. This structuring makes it easier to distinguish the scripts that we share and the local ones that pertain only to this repo.